src/sql/src/plan.rs

```diff
@@ -55,6 +56,8 @@ pub use self::expr::RelationExpr;
 pub use error::PlanError;
 // This is used by sqllogictest to turn SQL values into `Datum`s.
 pub use query::scalar_type_from_sql;
+// This is used by symbiosis to validate CREATE TABLE statements.
```

Comment:
@benesch is there a better way to expose this functionality to the symbiosis module?

Technically, symbiosis could ignore validation since the upstream request to postgres will fail anyway but on the other hand it does validate other invariants already.

Reply:
Ah yeah, symbiosis mode is long overdue to be deleted. It was created because back in the day we didn't support tables, and it persists because we still don't support `UPDATE` and `DELETE`. Once we do, we can delete all the nasty symbiosis stuff :). I've gone ahead and elided this bit from the patch, because symbiosis mode is only used in tests and therefore it only needs to do the bare minimum to make the tests pass. (Don't want to torture the APIs of other crates for symbiosis's purposes since its lifespan is so short.)